src/auto-evo/AutoEvoExploringTool.cs

```diff
@@ -0,0 +1,640 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using AutoEvo;
+using Godot;
+using HarmonyLib;
+
+public class AutoEvoExploringTool : NodeWithInput
```

Comment:
This is a pretty big feature and this is the main class, so could you add a brief summary here what the whole exploring tool is about?